docs/Contratos.html

```diff
@@ -0,0 +1,113 @@
+<!doctype html>
```

Comment:
Por convención, es preferible que los nombres de los archivos .html, .css, etc. etc. se haga en [kebab-case](https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841) (minúsculas, con separaciones con guión medio "-"), cambiar el nombre a contratos.html

Reply:
Corregido